certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -5094,7 +5094,7 @@
         "x-ms-api-annotation": {
           "family": "ParseDocument",
           "revision": 1,
-          "status": "Preview"
+          "status": "Production"
```

Comment:
Can you please let's know the reason for changing from Preview and Production.

Reply:
the connector is now available under GA. the action has been tested and should be moved to Production status

Reply:
Please refer your connector documentation link below, this change has been done already and moved to GA
https://learn.microsoft.com/en-us/connectors/pdf4meconnect/